clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -183,6 +183,9 @@ Changes in existing checks
   <clang-tidy/checks/modernize/use-override>` check to also remove any trailing
   whitespace when deleting the ``virtual`` keyword.
 
+- Improved :doc:`modernize-use-using <clang-tidy/checks/modernize/use-using>`
+  check by fixing false-negative in functions.
```

Comment:
maybe: `by adding support for detection of typedefs declared on function level.`